### PR TITLE
docs: added contributors

### LINKED ISSUE #3206: Add various acknowledgements
Using the @all-contributors  bot add various contributors to the CONTRIBUTORS.md  file

### PR DESCRIPTION
**Pull request by jonrkarr**
closes #3206

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -71,12 +72,221 @@
...
     }
   ],
   "contributorsPerLine": 7,
   "projectName": "biosimulations",
   "projectOwner": "biosimulations",
   "repoType": "github",
   "repoHost": "https://github.com",
-  "skipCi": true
+  "skipCi": true,
+  "commitConvention": "none"
```
Comment (bilalshaikh42) [inline .all-contributorsrc]: remove commitConvention line to keep semantic commits

**Review by bilalshaikh42 — CHANGES_REQUESTED** [review]

**Comment by jonrkarr** [comment]
@check-spelling-bot apply changes

File: .all-contributorsrc (hunk 1 above)
Comment (jonrkarr) [reply .all-contributorsrc]: Done. The CLI tool automatically changes this.

**Pull request merged**